src/main/java/net/minecraftforge/fml/common/registry/VillagerRegistry.java

```diff
@@ -133,8 +136,7 @@ public void registerVillagerId(int id)
      * @param villagerSkin
      */
     @SideOnly(Side.CLIENT)
-    @Deprecated // Doesn't work at all.
-    public void registerVillagerSkin(int villagerId, ResourceLocation villagerSkin)
+    private void registerVillagerSkin(int villagerId, ResourceLocation villagerSkin)
     {
```

Comment:
This is deprecated to be removed in 1.8.8 but still exists because we dont want to fire MissingMethodExceptions.